### DISCUSSION TITLE
We are looking for PS5 contributors

### DISCUSSION THREAD

**Discussion by ultimaweapon** [obhq/obliteration#1166] [Announcements]
Our current team (including me) currently focus on re-implementing the PS4 kernel. Recently a team of hacker successfully defeat the PS5 security and are able to dump its kernel. AFAIK the PS5 kernel should be similar to the PS4 so adding PS5 support to our kernel likely to be implementing additional code instead of a rework. So we are looking for people who have a hackable PS5 that also interested to work on this. You don't need to know low-level stuff or how to reverse engineering. We are happy to help you getting started with this. All you need is a hackable PS5 and proficient at least one programming language. It don't need to be a low-level language like C/C++ since you are going to use Rust.

The following are the roughly steps to start working if you are interested:

1. Dump the PS5 kernel. You may adapt our [Kernel Dumper](https://github.com/obhq/kernel-dumper) for this or using the other dumper.
2. Open the dump with disassembler of your choice. We recommend [Ghidra](https://github.com/NationalSecurityAgency/ghidra) since it is free and open-source and its decompiler is very good at dealing with binary produced by C language.
3. Make [OKF](https://github.com/obhq/kernel-framework) supports PS5.
4. Make [Firmware Dumper](https://github.com/obhq/firmware-dumper) support PS5.
5. Dump PS5 system files using Firmware Dumper.
6. Start working on Obliteration source.

If you have any questions or need help feel free to ask us here or on our [Zulip](https://obkrnl.zulipchat.com).

**Comment by Unfrgt10** [comment]
Here is something that might help
https://archive.org/details/prosperoupdate-devkit-0-830-020
